### ISSUE TITLE
Fedora 26 can not install

### ISSUE THREAD

**Issue by LinjingFcode** [MaartenBaert/ssr#553]
I had tried to install simple screen recorder through the code on "http://www.maartenbaert.be/simplescreenrecorder/#feedback-bugs-feature-requests" but it did not work, then i tried to download by the website and successfully downloaded, but the package can not be installed.

**Comment by Yadnyawalkya** [comment]
Yeah, same happened with me, after recording started black screen appears and only cursor get recorded.
I think it requires some package dependencies which are not yet ready for Fedora26.

**Comment by MaartenBaert** [comment]
I don't maintain the Fedora package (or even use Fedora), so I can't really help you with this, sorry.

Did you get some type of error message?
